ee/fleetctl/updates.go

```diff
@@ -171,15 +171,17 @@ func updatesRootsFunc(c *cli.Context) error {
 		return err
 	}
 
-	keys, err := repo.RootKeys()
+	meta, err := repo.GetMeta()
 	if err != nil {
-		return fmt.Errorf("get root metadata: %w", err)
+		return fmt.Errorf("get repo metadata: %w", err)
 	}
-
-	if err := json.NewEncoder(os.Stdout).Encode(keys); err != nil {
-		return fmt.Errorf("encode root metadata: %w", err)
+	rootMeta := meta["root.json"]
+	if rootMeta == nil {
+		return errors.New("missing root metadata")
 	}
 
+	fmt.Println(string(rootMeta))
```

Comment:
Remove?

Reply:
Noup, we want to print to stdout, in the removed changes you can see a marshal/encode to `os.Stdout`.